examples/logger.py

```diff
@@ -3,13 +3,33 @@
 """
 
 import time
+import logging
 from typing import Any, Optional, Tuple
 
 from PyQt5.QtCore import QObject, pyqtSlot, pyqtSignal
 from PyQt5.QtWidgets import QWidget, QVBoxLayout, QPushButton, QTextEdit, QLabel, QDialogButtonBox
 
 from qiwis import BaseApp
 
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+class LoggingHandler(QObject, logging.Handler):
+    """
+    Handler for logger, handle input log by emit
+    """
+    signal = pyqtSignal(str)
+    def __init__(self, slotfunc, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.signal.connect(slotfunc)
+
+    def emit(self, record):
+        """ 
+        emit input signal to connected function
+        """
```

Comment:
1. Do not break the line after the first `"""`.
2. Use a capital letter for the first character.
3. Use the 3rd person singular verbs (emits, not emit) for function/method docstrings. It is easy to think the subject of the sentence is "it" and you omit it.
4. Finish sentences with a period.
5. Write the docstring for arguments.

Following the [style guide](https://google.github.io/styleguide/pyguide.html), the docstring should be:
```python
        """Emits input signal to connected function.
        
        Args:
            record: The emitted log record.
        """
```

However, since this is _overriding_ the method of `logging.Handler`, we can write as (see also: #44):
```python
        """Overridden."""
```